cedar-drt/fuzz/fuzz_targets/formatter.rs

```diff
@@ -104,12 +105,31 @@ fn round_trip(p: &StaticPolicy) -> Result<StaticPolicy, parser::err::ParseErrors
         line_width: 80,
     };
     let mut uuids = Vec::new();
+    let commented = attach_comment(&p.to_string(), &mut uuids);
     let formatted_policy_str =
-        &policies_str_to_pretty(&attach_comment(&p.to_string(), &mut uuids), &config)
-            .expect("pretty-printing should not fail");
+        &policies_str_to_pretty(&commented, &config).expect("pretty-printing should not fail");
     // check if pretty-printing drops any comment
+    let mut formatted_policy_tail: &str = formatted_policy_str.as_str();
     for u in &uuids {
-        assert!(formatted_policy_str.contains(u), "missing comment: {}\n", u);
+        match formatted_policy_tail.split_once(u) {
+            Some((_, after_uuid)) => formatted_policy_tail = after_uuid,
+            None => {
```

Comment:
Nit: remove it?

Reply:
It's printed only once we know the test case is an error, so no reason to remove it, and it makes debugging failures much easier